features/cells/src/main/java/com/wire/android/feature/cells/ui/filter/FilterBottomSheet.kt

```diff
@@ -0,0 +1,223 @@
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.rotate
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import com.wire.android.feature.cells.R
+import com.wire.android.ui.common.bottomsheet.WireModalSheetLayout
+import com.wire.android.ui.common.bottomsheet.WireModalSheetState
+import com.wire.android.ui.common.bottomsheet.WireSheetValue
+import com.wire.android.ui.common.bottomsheet.rememberWireModalSheetState
+import com.wire.android.ui.common.button.WireButtonState
+import com.wire.android.ui.common.button.WirePrimaryButton
+import com.wire.android.ui.common.button.WireSecondaryButton
+import com.wire.android.ui.common.chip.WireFilterChip
+import com.wire.android.ui.common.colorsScheme
+import com.wire.android.ui.common.dimensions
+import com.wire.android.ui.common.preview.MultipleThemePreviews
+import com.wire.android.ui.theme.WireTheme
+import com.wire.android.ui.theme.wireTypography
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+
+@Composable
+fun FilterBottomSheet(
+    selectableTags: StateFlow<List<String>>,
+    selectedTags: StateFlow<List<String>>,
+    onApply: (List<String>) -> Unit,
+    onClearAll: () -> Unit,
+    onDismiss: () -> Unit,
+    sheetState: WireModalSheetState<Unit> = rememberWireModalSheetState<Unit>(WireSheetValue.Expanded(Unit))
+) {
+    WireModalSheetLayout(
+        onDismissRequest = onDismiss,
+        sheetState = sheetState
+    ) {
+        SheetContent(selectableTags, selectedTags, onApply, onClearAll)
+    }
+}
+
+@Composable
+private fun SheetContent(
+    selectableTags: StateFlow<List<String>>,
+    selectedTags: StateFlow<List<String>>,
+    onApply: (List<String>) -> Unit = {},
+    onClearAll: () -> Unit = {},
+) {
+    var isExpanded by remember { mutableStateOf(true) }
+
+    val transition = updateTransition(targetState = isExpanded)
+    val height by transition.animateDp { state ->
+        if (state) {
+            dimensions().spacing120x
+        } else {
+            dimensions().spacing52x
+        }
+    }
```

Comment:
Instead of hardcoding heights for both states and creating this transition, this expand/collapse animation could be done easier by just using `Modifier.animateContentSize()`:
```
Column(modifier = Modifier.animateContentSize()) {
    if (isExpanded) {
        LazyRow(...)
    }
}
```
 or wrapping the content with `AnimatedContent` like:
 ```
Column {
    AnimatedContent(isExpanded) { isExpanded ->
        if (isExpanded) {
            LazyRow(...)
        }
    }    
}
 ```